gunicorn/workers/gthread.py

```diff
@@ -36,10 +36,7 @@
     Python version.
     """)
 
-try:
-    from asyncio import selectors
-except ImportError:
-    from gunicorn import selectors
+import selectors
```

Comment:
One more comment: Just to be more safe, I think we can raise a more descriptive message in case the selectors module was not found (similar to what we already did for concurrent.futures):

```py
try:
    import selectors
except ImportError:
    try:
        import selectors34 as selectors
    except ImportError:
        raise RuntimeError(
            "You need to install the 'selectors34' package to use this worker "
            "with this Python version."
        )
```

Reply:
Or we can create the 19.x branch before merging this PR, I don't know.

Reply:
Good idea, added!